apps/mi/mi-adf-shir/image-policy.yaml

```diff
@@ -17,13 +19,15 @@ spec:
 apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
 metadata:
-  name: mi-adf-shir-dev-2022
+  name: mi-adf-shir-dev
```

Comment:
changed to dev instead of -2022 as each aks cluster can only have 1 active windows os at a time, so its sufficient to target per env rather than alo differentiate the image tags.